nemo_skills/dataset/ojbench/prepare.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+REPO_URL = "https://huggingface.co/datasets/He-Ren/OJBench_testdata"
+HF_TOKEN = os.environ.get("HF_TOKEN")
```

Comment:
please raise some meaningful error if this is not defined if HF_TOKEN is required